cdap-proto/src/main/java/co/cask/cdap/proto/profile/Profile.java

```diff
@@ -28,7 +28,7 @@
  * environment. A profile is identified by name and must be assigned a provisioner and its related configuration.
  */
 public class Profile {
-  public static final Profile DEFAULT = new Profile("default", "Runs programs locally on the cluster",
+  public static final Profile DEFAULT = new Profile("native", "Runs programs locally on the cluster",
```

Comment:
For consistency's sake, should this also be renamed to `NATIVE`?

Reply:
yeah, was in the middle of pushing the rename but you reviewed too fast :)